### ISSUE TITLE
Genymotion connection

### ISSUE THREAD

**Issue by jgsqware** [sorccu/docker-adb#2]
Hello,

i'm trying to use a adb container to connect to my genymotion emulator. 

Have you tried to do something like this?

Julien

**Comment by sorccu** [comment]
Nope, but let me know if you find a way. If it requires using adb connect,
make sure that your genymotion is exposing the port on the necessary
network interfaces.
